.changeset/common-lizards-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Added 13 new Playwright lint rules to the nursery (from eslint-plugin-playwright)
```

Comment:
We usually have one changeset per rule, but personally, I'm fine with having just one.

But it does at least need to contain the names of the rules added, and a short description of what they do.